docs/toc.js

```diff
@@ -2,327 +2,42 @@ module.exports = {
-      ]],
-      ['Context', ['context',
-        ['Get Context', 'get-context.md'],
-        ['Get All Contexts', 'get-contexts.md'],
-        ['Set Context', 'set-context.md'],
-      ]],
-      ['Interactions', ['interactions',
-        ['Mouse', ['mouse',
-          ['Move To', 'moveto.md'],
-          ['Click', 'click.md'],
-          ['Double Click', 'doubleclick.md'],
-          ['Button Down', 'button-down.md'],
-          ['Button Up', 'button-up.md'],
-        ]],
-        ['Touch', ['touch',
-          ['Single Tap', 'tap.md'],
-          ['Double Tap', 'double-tap.md'],
-          ['Move', 'move.md'],
-          ['Touch Down', 'touch-down.md'],
-          ['Touch Up', 'touch-up.md'],
-          ['Long Press', 'long-press.md'],
-          ['Scroll', 'scroll.md'],
-          ['Flick', 'flick.md'],
-          ['Multi Touch Perform', 'multi-touch-perform.md'],
-          ['Touch Perform', 'touch-perform.md'],
-        ]],
-        ['W3C Actions', 'actions.md'],
-      ]],
-      ['Web', ['web',
-        ['Window', ['window',
-          ['Set Window', 'set-window.md'],
-          ['Close Window', 'close-window.md'],
-          ['Get Handle', 'get-handle.md'],
-          ['Get Handles', 'get-handles.md'],
-          ['Get Title', 'title.md'],
-          ['Get Window Size', 'get-window-size.md'],
-          ['Set Window Size', 'set-window-size.md'],
-          ['Get Window Position', 'get-window-position.md'],
-          ['Set Window Position', 'set-window-position.md'],
-          ['Maximize Window', 'maximize-window.md'],
-        ]],
-        ['Navigation', ['navigation',
-          ['Go to URL', 'go-to-url.md'],
-          ['Get URL', 'url.md'],
-          ['Back', 'back.md'],
-          ['Forward', 'forward.md'],
-          ['Refresh', 'refresh.md'],
-        ]],
-        ['Storage', ['storage',
-          ['Get All Cookies', 'get-all-cookies.md'],
-          ['Set Cookie', 'set-cookie.md'],
-          ['Delete Cookie', 'delete-cookie.md'],
-          ['Delete All Cookies', 'delete-all-cookies.md'],
-        ]],
-        ['Frame', ['frame',
-          ['Switch to Frame', 'switch.md'],
-          ['Switch to Parent Frame', 'parent.md'],
-        ]],
-        ['Execute Async', 'execute-async.md'],
-        ['Execute', 'execute.md'],
-      ]],
+      ['Driver-specific commands', 'driver-commands.md'],
     ]],
-    ['Writing & Running Tests', ['writing-running-appium',
-      ['Running Tests', 'running-tests.md'],
+    ['The Server', ['server',
+      ['CLI Arguments', 'server-args.md'],
+      ['Driver and Plugin Arguments', 'extension-args.md'],
       ['Desired Capabilities', 'caps.md'],
       ['The --default-capabilities flag', 'default-capabilities-arg.md'],
-      ['Finding Elements', 'finding-elements.md'],
-      ['Touch Actions', 'touch-actions.md'],
-      ['CLI Arguments', 'server-args.md'],
       ['Server Security', 'security.md'],
-      ['Web/Web Views', ['web',
-        ['Mobile Web Testing', 'mobile-web.md'],
```

Comment:
How like this common-method/Tutorial/tips etc will be?
(Ideally) Move them into each driver repository...? (I'd like to know current idea.)

Reply:
Yep, exactly, move them into each driver repository. I think each driver should have its own docs generation and docs site, or at least links between markdown files to start, so that all of this information is not lost, just moved to the appropriate drivers.